Plugins/SQL/SQL.cpp

```diff
@@ -228,12 +228,12 @@ Events::ArgumentStack SQL::OnExecutePreparedQuery(Events::ArgumentStack&&)
         if (m_target->GetAffectedRows() >= 0)
         {
             // this was not a result set type query
-            LOG_INFO("Successful SQL query. Query ID: '%i', Query: '%s', Rows affected: '%u'.",
+            LOG_DEBUG("Successful SQL query. Query ID: '%i', Query: '%s', Rows affected: '%u'.",
```

Comment:
This shouldn't be DEBUG. At debug level, we already get all the info and much more. If it is too verbose for INFO, then just remove entirely.